docs/text_summaries.ipynb

```diff
@@ -162,16 +169,43 @@
         "id": "rngALbRogXe6"
       },
       "source": [
-        "Now, use TensorBoard to examine the image. Wait a few seconds for the UI to spin up."
+        "Now, use TensorBoard to examine the text. Wait a few seconds for the UI to spin up."
       ]
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 6,
       "metadata": {
         "id": "T_X-wIy-lD9f"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "\n",
+              "      <iframe id=\"tensorboard-frame-9381929c3767b97b\" width=\"100%\" height=\"800\" frameborder=\"0\">\n",
```

Comment:
I'm not sure we want to cache the output for this iframe?  I wouldn't expect it will render anything useful (outside the tensorflow.org site, where I think it gets replaced by a static image), so it seems better to omit IMO.

Ditto elsewhere below where we have the iframe.

Reply:
Removed, 

Aside, I've been editing this file in the jupyter lab UI.  Is there an easy way to drop the cell output there, or do I need to edit somewhere else?

Reply:
I don't know whether there's a way to do that in the jupyter lab UI; I am not very familiar with it.  But you can always just edit the JSON file.